package.json

```diff
@@ -44,12 +44,9 @@
     "expo-splash-screen": "0.6.1",
     "expo-status-bar": "1.0.2",
     "expo-web-browser": "8.5.0",
-    "front-matter": "3.1.0",
     "glob": "7.1.6",
-    "husky": "4.2.5",
     "i18n-js": "3.5.1",
     "lodash.throttle": "4.1.1",
-    "markdown-it": "10.0.0",
     "moment": "2.24.0",
     "paths-js": "0.4.10",
     "prop-types": "15.7.2",
```

Comment:
prop-types can be deleted, not use anywhere (legacy)

Reply:
Done